ldap_jwt_auth/auth/jwt_hanlder.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
Name should be `jwt_handler.py`

Reply:
Addressed in https://github.com/ral-facilities/ldap-jwt-auth/pull/7/commits/3fe572b003b5c1db3886647a299ca836093a4639